### DISCUSSION TITLE
About this project

### DISCUSSION THREAD

**Discussion by domneedham** [domneedham/pico-clock-green-python#1] [Announcements]
This is an implementation of the Pico-Clock-Green written in MicroPython.

A big thanks to Malcolm Holmes (@malcolmholmes) and contributors who worked on the first version of this project (and where much of the code has come from / inspired). This repository will be a maintained fork by me, with lots of new features to be added. 

For visibility, this project was forked from https://github.com/malcolmholmes/pico-clock-green-python.

**Comment by malcolmholmes** [comment]
Am more than happy for this version to replace mine on the WaveShare site, assuming it adds meaningful new features. I'm not actively developing it right now, and don't have a setup for testing PRs anyhow.

**Reply by domneedham** [reply]
My version is added alongside yours so both are there on the wiki. The plan is to add new features, particularly around wireless features for the Pico W. 

I presume there won't be that many PRs coming in anyway, so manual testing for now will do. Though I will have to look at proper testing down the line. 

I wanted to make sure the visibility of your work was still there in the repository (hence your mention in this) as the code still relies on a lot of your work. I have added some quality of life stuff though, such as using the second core and uasyncio for improved scheduling :)
